buildchain/buildchain/vagrant.py

```diff
@@ -40,16 +42,53 @@ def mkdir_dot_vagrant() -> None:
         'uptodate': [True],
     }
 
+def task_vagrantupnoprov() -> types.TaskDict:
+    """Run `vagrant up` without provisioning a development environment."""
+    vagrantupnoprov = [config.VAGRANT, 'up']
+    vagrantupnoprov.extend(map(shlex.quote,
+        config.VAGRANT_UP_ARGS + tuple(('--no-provision',))))
```

Comment:
Why not `+ ['--no-provision']`?

Reply:
Because `config.VAGRANT_UP_ARGS` is a tuple, and we cannot concatenate list to tuples... (tried it in the 1st place ;)

Reply:
Oh, indeed!

Maybe it can be more readable (the single-element tuple with its trailing comma is always tricky…) by doing it in two steps:

```Python
    vagrantupnoprov.extend(map(shlex.quote, config.VAGRANT_UP_ARGS))
    vagrantupnoprov.append('--no-provision')
```